src/ape_compile/__init__.py

```diff
@@ -5,10 +5,30 @@
 class EvmVersion(ConfigEnum):
     constantinople = "constantinople"
     istanbul = "istanbul"
+    paris = "paris"
+    shanghai = "shanghai"
 
 
 class Config(PluginConfig):
-    evm_version: EvmVersion = EvmVersion.istanbul
+    evm_version: EvmVersion = EvmVersion.shanghai
```

Comment:
i never knew this existed and im pretty sure this is not being honored or used anywhere. all compiled that are evm have their own settings here

Reply:
Was envisioning that this would work as an more global setting to compiler plugins

Reply:
honestly, this is also kind of weird with the plugin system because non-EVM compilers would never use this, so I agree with removing it now as the right outcome

Reply:
k it gone